core/src/main/java/bisq/core/btc/nodes/LocalBitcoinNode.java

```diff
@@ -229,6 +231,7 @@ public void onSuccess(Peer peer) {
                 peerVersionMessageFuture.set(peer.getPeerVersionMessage());
             }
             public void onFailure(Throwable thr) {
+                ;
```

Comment:
```suggestion
                // No action
```

Reply:
I think a comment might work, codacy is complaining about the lone semicolon, which does look odd.